index.bs

```diff
@@ -361,13 +361,13 @@ one line except when a line break is definitely necessary.</p>
  <pre>
  WEBVTT
 
- 00:00:00.000 --> 00:00:04.000 position:10%,start align:left size:35%
+ 00:00:00.000 --> 00:00:04.000 position:10%,left align:left size:35%
```

Comment:
For vertical cues, position refers to a vertical offset. Then, start and end don't refer to left and right, but to top and bottom. I'm reluctant to change these keywords because of that.